src/System.ValueTuple/tests/ExtensionsTests.cs

```diff
@@ -5,13 +5,12 @@
 using System;
 using System.Text;
 using Xunit;
-using static ValueTupleTests;
 
 public class ExtensionsTests
```

Comment:
Sure. Added `namespace System.ValueTuple.Tests` to all three test files.
I'll update my NetFxDev1 shelveset accordingly.